orchestration/tests/test_task_dispatcher_fix.py

```
@@ -573,6 +573,143 @@ def test_branch_exists_logs_exception_details(self):
+            patch("orchestration.task_dispatcher.subprocess.run") as mock_run,
+            patch("orchestration.task_dispatcher.logger") as mock_logger,
+        ):
+            # First call is _detect_pr_context → _find_recent_pr which calls subprocess
+            # The gh pr view call is the one that should raise
+            def run_side_effect(cmd, **kwargs):
+                if "pr" in cmd and "view" in cmd:
+                    raise OSError("gh not found")
+                return MagicMock(returncode=0, stdout="", stderr="")
+
+            mock_run.side_effect = run_side_effect
+            agents = self.dispatcher.analyze_task_and_create_agents(task)
```

Ruff (0.15.0)

[error] Local variable `agents` is assigned to but never used

Remove assignment to unused variable `agents`

(F841)